doc/source/quickstart/play.rst

```diff
@@ -0,0 +1,34 @@
+How to play
+===========
+
+Goal
+----
+
+The goal is to get points by hitting notes.
+
+Press and hold the appropriate frets when notes appear and tap the pick button
+when the notes hit the row of keys at the bottom of the screen. Hold the frets down for long notes.
+
+
+Keys
+----
+
+Thoses keys are default keys for a keyboard (the same as `FretsOnFire <http://fretsonfire.sourceforge.net/documentation/howto/>`_) and can be changed.
+
+In the menu
++++++++++++
+
+- **Arrow keys**: change menu selection
+- **Enter**: validate
+- **Escape**: cancel.
+
+
+In the game
++++++++++++
+
+- **F1 - F5**: rets one to five
+- **F6 - F10**: solo frets one to five
+- **Enter / Shift right**: pick
+- **Page down**: star power
+- **Page up**: whammy
+- **Alt + Enter**: switch to fullscreen.
```

Comment:
Just fyi i think alt - enter is disabled on windows at least you might want to double check that though.

Reply:
I found that for Windows: https://github.com/fofix/fofix/blob/master/fofix/core/GameEngine.py#L424. I can't test it right now, but I think the doc need only a precision about restarting the game